[Github] Update PR template (#16823)
microsoft/PowerToys: .github/pull_request_template.md

**What is this about:**

**What is included in the PR:** 

**How does someone test / validate:** 

## Quality Checklist

- [ ] **Linked issue:** #xxx
- [ ] **Communication:** I've discussed this with core contributors in the issue. 
- [ ] **Tests:** Added/updated and all pass
- [ ] **Installer:** Added/updated and all pass

> - [ ] **Localization:** All end user facing strings can be localized

Author: Add again

   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries and localization folder
   - [ ] [YML for build pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects

> ### Note
> Please ensure that all strings can be localized and the [user docs](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) are updated or you have created an issue for the user docs.

Author: ```suggestion
### End-user docs
Please ensure that the [end-user documentation](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) is up to date or you have created an issue for the user docs.
```

Author: @jaimecbernardo
Are you okay with this phrasing? Then I going to push a commit this evening (German time).

Reviewer: "for the outstanding tasks." sounds weird. How do you feel about "with the remaining tasks."?
Rest LGTM!

**What is this about:**

**What is included in the PR:** 

**How does someone test / validate:** 

## Quality Checklist

- [ ] **Linked issue:** #xxx

> - [ ] **Communication:** I've discussed this with core contributors in the issue. 

Reviewer 2: I want this, this was one of the reasons why we added this.  It is a reminder to the person that all parties agreed on what was planned to be done.

## Quality Checklist

- [ ] **Linked issue:** #xxx
- [ ] **Communication:** I've discussed this with core contributors in the issue.
- [ ] **Code quality:**
   - [ ] **Tests:** Added/updated and all pass
   - [ ] **.Net Analyzer:** Enabled and warnings fixed for new projects (See [dev docs](/doc/devdocs/readme.md#rules) for more infos.)
- [ ] **Installer:** Added/updated and all pass
- [ ] **Localization:** All end user facing strings can be localized
- [ ] **Dev docs:** Added/updated
- [ ] **User docs**:
   - [ ] No updates required
   - [ ] New PR on [Microsoft Docs](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) repository: #xxx

>    - [ ] New issue to do work later: #xxx

Reviewer 3: ```suggestion
   - [ ] New issue to do work later: #xxx (Assignee: @ xxx)
```

- [ ] **Binaries:** Any new files are added to WXS / Signing pipeline / YML
   - [ ] No new binaries
   - [ ] [JSON for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ESRPSigning_core.json) for new binaries
   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries

>    - [ ] [YML for build pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects

Author: But without this the unit test project isn't executed. And I personally had to search and find out why.

>    - [ ] [YML for build pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects

Reviewer 2: I think that them could be discussed in the unit test line

>    - [ ] [YML for build pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects

Author: Do you want me to move it?
